data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3360 @@
+			action
+				"reputation: Wanderer" = 1
+			label neutral
+			`	"Do scout all of their systems for us, and bring back any information you may gather."`
+			choice
+				`	"I guess I will try to land on their planets."`
+					accept
+				`	"I am not interested in being able to communicate with them."`
+			`	"Well if you want to work with us then you will have to set that preferance aside. And do not forget the translator is for communicating with us as well. Some among the most conversative of us despise speaking in your language... This will be a token of good faith on your part."`
+			choice
+				`	"I guess I will try to land on their planets."`
+					accept
+				`	"I will not work for you anymore then."`
+					decline
+	
+	waypoint "Chy'chra"
+	waypoint "Pik'ro'iyak"
+	waypoint "Ik'kara'ka"
+	waypoint "Chirr'ay'akai"
+	waypoint "Sich'ka'ara"
+	waypoint "Kiro'ku"
+	waypoint "Ka'pru"
+	waypoint "Ap'arak"
+	waypoint "Ka'ch'chrai"
+	waypoint "Ek'kek'ru"
+	waypoint "Si'yak'ku"
+	waypoint "Iyech'yek"
+	waypoint "Kiru'kichi"
+	waypoint "Es'sprak'ai"
+	
+	on waypoint
+		dialog `You have scouted all of the Wanderer's systems. If you can somehow speak Hai already, it is time to come back to <destination>, for a reward.`
+	to complete
+		has "language: Wanderer"
+	on complete
+		log `The Unfettered asked me to be able to speak like a True Hai, because that is the language in which I should make demands to the Unfettered later.`
+		log "Wanderer" `The Wanderers are a birdlike specie that seems weak and is said to inhabit the old lands of the Unfettered Hai.`
+		payment 1000000
+		dialog `You find the herald back. "Now you can speak to us like a True Hai, finally. By respect to you, we will keep speaking to you in your language. Except for some maybe... You only should use the translator to speak and understand our elders, or at least some of them. They will probably do as if they noticed nothing, but trust me, they care. Oh and here are <payment> for the intel, we could not have gotten it so easily given they shoot our ships on sight."`
+
+
+
+mission "Unfettered: Bring Detainer"
+	landing
+	name `Recover a detainer`
+	description `He Yi has asked you to earn a translation device by bringing back a powerful alien weapon, said to be able to neutralize the engines of its target. You should probably look around the sestor automaton's space.`
+	source "Warfeed"
+	to offer
+		has "Unfettered: Know Your Enemy: active"
+		has "First Contact: Wanderer: offered"
+		not "language: Wanderer"
+	on offer
+		conversation
+			`You go find He Yi in his war room, he does not seem too surprised to see you. "I expected you to come here. You most want one of us to act as 'diplomat' with the aliens from the North."`
```

Comment:
Not entirely sure what can be done about it, other than a -ton- of branching that could end up getting kind of messy, but just wanted to bring up that I got this mission which mentions He Yi by name before he actually introduced himself in the other mission.

Reply:
Multiple conditions can be used for only one branching I think (and hope)
I'll just make smth where you are guided to him. But isn't shipyard intel before this?

Reply:
I think it's simpler if I just make it happen later for now, but I'll change it up at some point to make He Yi more mysterious, in the dark, before he shows his face, and, finally, his name

Reply:
Happening later is actually very good! You have access to as many SB as you want, meaning you can fight the automatons if you want to.
About He Yi I had forgotten every time you meet him, it is in the spaceport, so I'll just give him a hood at first.